lib/Config/Migrator.php

```diff
@@ -448,6 +449,26 @@ function statisticsForms() {
     return $this->sqlify(__FUNCTION__, $attributes);
   }
 
+  function statisticsWoocommerceOrders() {
+    $attributes = array(
+      'id int(11) unsigned NOT NULL AUTO_INCREMENT,',
+      'newsletter_id int(11) unsigned NOT NULL,',
+      'subscriber_id int(11) unsigned NOT NULL,',
+      'queue_id int(11) unsigned NOT NULL,',
+      'click_id int(11) unsigned NOT NULL,',
+      'order_id bigint(20) unsigned NOT NULL,',
+      'order_currency varchar(10) NOT NULL,',
+      'order_price_total float NOT NULL COMMENT "With shipping and taxes in order_currency",',
+      'created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,',
```

Comment:
Shouldn't we also have `updated_at`? AFAIK all the other models have it.

Reply:
GitHub is weird. It didn't show me this comment until I went through commits one-by-one 😞Fixed.